editor/editor_spin_slider.cpp

```diff
@@ -154,7 +154,7 @@ void EditorSpinSlider::_grabber_gui_input(const Ref<InputEvent> &p_event) {
 
 void EditorSpinSlider::_notification(int p_what) {
 
-	if (p_what == MainLoop::NOTIFICATION_WM_FOCUS_OUT || p_what == MainLoop::NOTIFICATION_WM_FOCUS_OUT) {
```

Comment:
Maybe it was meant to be `NOTIFICATION_WM_FOCUS_IN`? I doubt reduz would have gone through the pain of a copy paste and `||` condition if it wasn't intended.